build-aux/generate.mk

```diff
@@ -419,12 +426,13 @@ $(OSS_HOME)/manifests/emissary/ambassador.yaml: $(OSS_HOME)/k8s-config/create_ya
 
 #
 # Generate report on dependencies
+# We exclude pkg_resources because it its inclusion seems to be a bug
+# (cf https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command)
+$(OSS_HOME)/build-aux/pip-show.txt: setup-diagd python-setup
+	source venv/bin/activate && pip freeze --exclude-editable | grep -v pkg_resources | cut -d= -f1 | xargs pip show > $@
```

Comment:
As much as I hate it, as long as the builder shell is how things actually get built, this belongs in the builder shell. `pip freeze` takes in to account the host platform. You'll get different results on different distros.

Reply:
(one of the reasons I'm cloning it in Go :) so you can get reproducible results anywhere)

Reply:
Ew. But yeah, OK.